### PR TITLE
Works on replace MAIN_DB_PREFIX by $db->prefix()

### PR DESCRIPTION
**Pull request by hregis**

### CODE REVIEW

**Comment by hregis** [comment]
@eldy this PR is ready
- change MAIN_DB_PREFIX by $db->prefix() for "product" classes and /core/class/
- lots of fixes for Travis compatibility

File: htdocs/core/class/infobox.class.php [hunk 1]
```diff
@@ -83,15 +83,15 @@ public static function getListOfPagesForBoxes()
...
+	 *  @param	DoliDB		$db				Database handler
 	 *  @param	string		$mode			'available' or 'activated'
 	 *  @param	int			$zone			Name or area (-1 for all, 0 for Homepage, 1 for Accountancy, 2 for xxx, ...)
 	 *  @param  User|null   $user	  		Object user to filter
 	 *  @param	array		$excludelist	Array of box id (box.box_id = boxes_def.rowid) to exclude
 	 *  @param  int         $includehidden  Include also hidden boxes
 	 *  @return array       	        	Array of boxes
 	 */
-	public static function listBoxes($dbs, $mode, $zone, $user = null, $excludelist = array(), $includehidden = 1)
+	public static function listBoxes($db, $mode, $zone, $user = null, $excludelist = array(), $includehidden = 1)
```
Comment (eldy) [inline htdocs/core/class/infobox.class.php:94]: Because the method is static, we used the var $dbs here to avoid conflict and overwritting global $db.

Comment (hregis) [reply htdocs/core/class/infobox.class.php]: @eldy ok thank you, i send a fix

**Comment by hregis** [comment]
@eldy ping

**Pull request merged**